frontend/src/components/ui/button.tsx

```diff
@@ -14,7 +14,7 @@ const buttonVariants = cva(
         default:
           "bg-primary text-primary-foreground shadow hover:bg-primary/90",
         destructive:
-          "bg-destructive text-destructive-foreground shadow-sm hover:bg-destructive/90",
+          "border border-destructive text-destructive shadow-sm hover:text-destructive/90",
```

Comment:
I don't think it's intentional here to change the default `destructive` style? @stackingsaunter is this a new `destructive-outline` variant?

CC @reneaaron 


Also now there's an inconsistency here with `Button` and `LoadingButton` (maybe the way we implemented LoadingButton is not good, internally it should probably render a Button?).

Reply:
added new variant desctructive_outline. shadcn variant if defined with hypen. hypen will be interpreted as substraction operation. so i defined destructive_outline